common/djangoapps/course_modes/models.py

```diff
@@ -375,6 +402,132 @@ def min_course_price_for_currency(cls, course_id, currency):
         modes = cls.modes_for_course(course_id)
         return min(mode.min_price for mode in modes if mode.currency == currency)
 
+    @classmethod
+    def enrollment_mode_display(cls, mode, verification_status):
+        """ Select appropriate display strings and CSS classes.
+
+            Uses mode and verification status to select appropriate display strings and CSS classes
+            for certificate display.
+
+            Args:
+                mode (str): enrollment mode.
+                verification_status (str) : verification status of student
+
+            Returns:
+                dictionary:
+        """
+
+        # import inside the function to avoid the circular import
+        from student.helpers import (
+            VERIFY_STATUS_NEED_TO_VERIFY,
+            VERIFY_STATUS_SUBMITTED,
+            VERIFY_STATUS_APPROVED
+        )
+
+        show_image = False
+        image_alt = ''
+
+        if mode == "verified":
+            if verification_status in [VERIFY_STATUS_NEED_TO_VERIFY, VERIFY_STATUS_SUBMITTED]:
+                enrollment_title = _("Your verification is pending")
+                enrollment_value = _("Verified: Pending Verification")
+                show_image = True
+                image_alt = _("ID verification pending")
+            elif verification_status == VERIFY_STATUS_APPROVED:
+                enrollment_title = _("You're enrolled as a verified student")
+                enrollment_value = _("Verified")
+                show_image = True
+                image_alt = _("ID Verified Ribbon/Badge")
+            else:
+                enrollment_title = _("You're enrolled as an honor code student")
+                enrollment_value = _("Honor Code")
+        elif mode == "honor":
+            enrollment_title = _("You're enrolled as an honor code student")
+            enrollment_value = _("Honor Code")
+        elif mode == "audit":
+            enrollment_title = _("You're auditing this course")
+            enrollment_value = _("Auditing")
+        elif mode in ["professional", "no-id-professional"]:
+            enrollment_title = _("You're enrolled as a professional education student")
+            enrollment_value = _("Professional Ed")
+        else:
+            enrollment_title = ''
+            enrollment_value = ''
+
+        return {
+            'enrollment_title': unicode(enrollment_title),
+            'enrollment_value': unicode(enrollment_value),
+            'show_image': show_image,
+            'image_alt': unicode(image_alt),
+            'display_mode': cls._enrollment_mode_display(mode, verification_status)
+        }
+
+    @staticmethod
+    def _enrollment_mode_display(enrollment_mode, verification_status):
+        """
+         return the of mode after checking enrollment mode and status.
```

Comment:
This is a nitpick, but this docstring and the ones below aren't consistent with the Google ["Napoleon"](http://google-styleguide.googlecode.com/svn/trunk/pyguide.html#Comments) style we try to adhere to for comments. Could you please make docstrings like this one consistent? That is, put a short description on the same line as the opening blockquote, and a more detailed explanation underneath, if necessary:
```
"""Select appropriate display strings and CSS classes.

Uses mode and verification status to select appropriate display strings and CSS classes for certificate display.
```